server/datastore/mysql/policies.go

```diff
@@ -62,10 +62,13 @@ func (ds *Datastore) PolicyByName(ctx context.Context, name string) (*fleet.Poli
 		fmt.Sprint(`SELECT `+policyCols+`,
 		COALESCE(u.name, '<deleted>') AS author_name,
 		COALESCE(u.email, '') AS author_email,
-		(select count(*) from policy_membership where policy_id=p.id and passes=true) as passing_host_count,
-		(select count(*) from policy_membership where policy_id=p.id and passes=false) as failing_host_count
+		COALESCE(ps.passing_host_count, 0) as passing_host_count,
+		COALESCE(ps.failing_host_count, 0) as failing_host_count
 		FROM policies p
 		LEFT JOIN users u ON p.author_id = u.id
+		LEFT JOIN policy_stats ps ON p.id = ps.policy_id
+			AND ((p.team_id IS NULL AND ps.inherited_team_id = 0)
+				OR (p.team_id IS NOT NULL AND ps.inherited_team_id = p.team_id))
```

Comment:
Is it possible to have a policy with the same name in two different teams?

Reply:
I confirmed you cannot have duplicate named policies between teams or globally.  this is enforced by a database table constraint ```  UNIQUE KEY `idx_policies_unique_name` (`name`)```

Reply:
Got it.
Thanks for checking.